PluginManager/po/it-local.po

```diff
@@ -0,0 +1,265 @@
+msgid "Version"
+msgstr "Versione"
+
+#: PluginManager/PluginManager.py:288
+msgid "Authors"
+msgstr "Autori"
+
+#: PluginManager/PluginManager.py:289
+msgid "Email"
+msgstr ""
+
+#: PluginManager/PluginManager.py:290
+msgid "Filename"
+msgstr "Nome del file"
+
+#: PluginManager/PluginManager.py:291
+msgid "Location"
+msgstr "Posto"
+
+#: PluginManager/PluginManager.py:296
+msgid "Loaded"
+msgstr "Caricato"
+
+#: PluginManager/PluginManager.py:299
+msgid "Hidden"
+msgstr "Nascosto"
+
+#: PluginManager/PluginManager.py:304 PluginManager/PluginManager.py:699
+msgid "Failed"
+msgstr "Fallito"
+
+#: PluginManager/PluginManager.py:377
+msgid "Installation Errors"
+msgstr "Errori di installazione"
+
+#: PluginManager/PluginManager.py:378
+msgid "The following addons had errors: "
+msgstr "Gli addons seguente presentano errori: "
+
+#: PluginManager/PluginManager.py:400
+msgid "Error"
+msgstr "Errore"
+
+#: PluginManager/PluginManager.py:401
+#, python-format
+msgid "Error removing the '%s' directory, The uninstall may have failed"
+msgstr ""
+"Errore nella cancellazione della cartella '%s'. La disinstallazione potrebbe "
+"non essere andata a buon fine"
+
+#: PluginManager/PluginManager.py:429
+msgid "Type"
+msgstr "Tipo"
+
+#: PluginManager/PluginManager.py:438
+msgid "Status"
+msgstr "Stato"
+
+#: PluginManager/PluginManager.py:441
+msgid ""
+"'*' items are supplied by 3rd party authors,\n"
+"<s>strikeout</s> items are hidden"
+msgstr ""
+"'*' elementi fornite da autori di terze parti,\n"
+"<s>elementi barrati</s> sono nascosti"
+
+#: PluginManager/PluginManager.py:449
+msgid "Name"
+msgstr "Nome"
+
+#: PluginManager/PluginManager.py:472
+msgid "Info"
+msgstr "Informazioni"
+
+#: PluginManager/PluginManager.py:475 PluginManager/PluginManager.py:586
+msgid "Hide"
+msgstr "Nascondi"
+
+#: PluginManager/PluginManager.py:478 PluginManager/PluginManager.py:600
+msgid "Install"
+msgstr "Installa"
+
+#: PluginManager/PluginManager.py:481
+msgid "Edit"
+msgstr "Modifica"
+
+#: PluginManager/PluginManager.py:483
+msgid "Load"
+msgstr "Carica"
+
+#: PluginManager/PluginManager.py:494
+msgid "Show hidden items"
+msgstr "Mostra gli elementi nascosti"
+
+#: PluginManager/PluginManager.py:502
+#, fuzzy
+msgid "Show Built-in items"
+msgstr "Mostra gli elementi interni"
+
+#: PluginManager/PluginManager.py:509
+msgid "* indicates 3rd party addon"
+msgstr "* indica gli addons di terze parti"
+
+#: PluginManager/PluginManager.py:514
+msgid "Plugins"
+msgstr ""
+
+#: PluginManager/PluginManager.py:584
+msgid "Unhide"
+msgstr "Mostra"
+
+#: PluginManager/PluginManager.py:603
+msgid "Uninstall"
+msgstr "Disinstalla"
+
+#: PluginManager/PluginManager.py:613
+#, fuzzy
+msgid "Checking Addons Failed"
+msgstr "Il controllo degli addons è fallito"
+
+#: PluginManager/PluginManager.py:614
+#, fuzzy
+msgid "The addon repository appears to be unavailable. Please try again later."
+msgstr ""
+"Il database degli addons non è disponibile. Si prega di riprovare più tardi."
```

Comment:
see above (also, I wouldn't use "database" for "repository", the meaning of the latter is more specific)